### PR TITLE
Adjust chat input font size

### LINKED ISSUE #5980: Update the font size of the text found in chat input to match the text that's display in the chat area
Text found in the chat area (after it's rendered) is 14.4px. In the chat input, the text is 16px. 

screen shot 2018-08-14 at 2 25 32 pm

### PR DESCRIPTION
**Pull request by KDSBrowne**
closes #5980

This pr also does a few more things.
1) fixes the userlist not sorting emoji's.
2) rename some items missing from #5973.
3) moves the 'clear status' position higher in the user-item menu.

### CODE REVIEW

**Comment by Tainan404** [comment]
I noticed two inconsistencies:
- When html5 reload, only html5 keeps with emoji
- Flash users aren't showed in html5 client.

html5-flash-inconcistency

File: bigbluebutton-html5/imports/ui/components/user-list/service.js [hunk 1]
```diff
@@ -54,15 +54,15 @@ const sortUsersByEmoji = (a, b) => {
...
     if (a.emoji.changedAt < b.emoji.changedAt) {
       return -1;
     } else if (a.emoji.changedAt > b.emoji.changedAt) {
       return 1;
     }
-  } else if (emojiA && emojiA !== EMOJI_STATUSES.none) {
+  } else if (emojiA && emojiA !== 'none') {
     return -1;
-  } else if (emojiB && emojiB !== EMOJI_STATUSES.none) {
+  } else if (emojiB && emojiB !== 'none') {
```
Comment (oswaldoacauan) [inline bigbluebutton-html5/imports/ui/components/user-list/service.js]: Why you removed the constant?

File: bigbluebutton-html5/imports/ui/stylesheets/variables/typography.scss [hunk 2]
```diff
@@ -4,7 +4,7 @@ $font-family-monospace:   Menlo, Monaco, Consolas, "Courier New", monospace !def
 
 $font-family-base:        $font-family-sans-serif !default;
 
-$font-size-base:          1rem !default;
+$font-size-base:          0.9rem !default;
```
Comment (oswaldoacauan) [inline bigbluebutton-html5/imports/ui/stylesheets/variables/typography.scss]: Does this font size complains with the WCAG 2.0 AA?
Changing this value will impact in all components, not only the chat input as requested in #5980.

File: bigbluebutton-html5/imports/ui/components/user-list/service.js (hunk 1 above)
Comment (KDSBrowne) [reply bigbluebutton-html5/imports/ui/components/user-list/service.js]: EMOJI_STATUSES.none doesn't exist, was removed so the 'clear status' option wouldn't appear in the list of available status.

**Pull request merged**